tests/helpers.py

```diff
@@ -83,6 +83,13 @@ def wait_for_resource_state(resource, state, timeout=60):
             reached the desired state
 
     """
+    if (
```

Comment:
Is this really necessary? This is just to get the status. No other operation is done here.

Reply:
Lets keep it for few runs, no harm. The accidental delete is more pain.